docs/Troubleshooting.md

```diff
@@ -0,0 +1,177 @@
+# Troubleshooting Guide for OMS Agent for Linux
+The following document provides a quick series of steps and procedures to diagnose and troubleshoot common issues.
+If none of these steps work for you the following channels for help are also available
+* Customers with Premier support can log a support case via [Premier](https://premier.microsoft.com/)
+* Customers with Azure support agreements can log support cases [in the Azure portal](https://manage.windowsazure.com/?getsupport=true)
+* File a [GitHub Issue](https://github.com/Microsoft/OMS-Agent-for-Linux/issues)
+* Feedback forum for ideas and bugs [http://aka.ms/opinsightsfeedback](http://aka.ms/opinsightsfeedback)
+
+# Table of Contents
+- [Important Log Locations](Important Log Locations)
```

Comment:
Please look at all in-document links in this document to insure they all work, thanks.